next.config.js

```diff
@@ -1,2 +1,6 @@
 const withSass = require('@zeit/next-sass')
-module.exports = withSass({})
+module.exports = withSass({
+  env: {
+    SERVER_URL: 'https://c0d3.com/signin'
```

Comment:
It's not good practice to expose your `SERVER_URL` in the config file. It should be loaded from an environmental variable like `process.env.SERVER_URL`